convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -39,12 +40,25 @@ def run(self):
                 "Skipping the check because there are no publicly available %s %d.%d repositories available."
                 % (system_info.name, system_info.version.major, system_info.version.minor)
             )
+            self.add_message(
+                level="INFO",
+                id="PACKAGE_UPDATES_CHECK_SKIP_NO_PUBLIC_REPOSITORIES",
+                message=(
+                    "Skipping the check because there are no publicly available %s %d.%d repositories available."
+                    % (system_info.name, system_info.version.major, system_info.version.minor)
+                ),
+            )
```

Comment:
Please, extract the raw message here and place it into a `message` variable, so it can be used in both the add_message and the logger call above.